helm-charts/aerospike-cluster/templates/aerospike-cluster-rbac.yaml

```diff
@@ -9,6 +9,12 @@ metadata:
     app: {{ template "aerospike-cluster.commonName" . }}
     chart: {{ .Chart.Name }}
     release: {{ .Release.Name }}
+{{- with .Values.imagePullSecrets }}
+imagePullSecrets:
+  {{- range . }}
+  - name: {{ . }}
+  {{- end }}
+{{- end }}
```

Comment:
Can you add a commented out entry for `imagePullSecrets` in `values.yaml` with an example?

Reply:
Also document the configuration in README.